### PR TITLE
Pre-hackage cleanup

### PR DESCRIPTION
**Pull request by jonascarpay**

### CODE REVIEW

File: aeson-commit.cabal [hunk 1]
```diff
@@ -1,56 +1,46 @@
...
-category:            Web
-build-type:          Simple
-cabal-version:       >=1.10
-extra-source-files:  README.md
+cabal-version:      >=1.10
+name:               aeson-commit
+version:            1.0
+license:            BSD3
+copyright:          2019 Cross Compass Ltd.
+maintainer:         viktor.kronvall@cross-compass.com
```
Comment (considerate) [inline aeson-commit.cabal]: I think you should be a maintainer as well @jonascarpay.

File: src/Data/Aeson/Commit.hs [hunk 2]
```diff
@@ -8,19 +8,22 @@ import           Data.Aeson.Types
...
--- | A parser that has _two_ failure modes; the 'ExceptT' or in the underlying 'Parser'.
---   The alternative instance only recovers from failures in the `ExceptT`.
---   This means that, as soon as we successfully construct a 'Right' value, the 'Alternative' considers the 'Commit' a success, even though the inner 'Parser' can still fail.
+-- | A 'Parser' that has _two_ failure modes; recoverable and non-recoverable.
+--   The default, recoverable failure is the equivalent to aeson's default 'Parser' behavior.
+--   The non-recoverable failure mode is used to commit to a branch; to commit means that every subsequent failure is non-recoverable.
 --
---   The 'Void' guarantees that that parser contains an error value.
+--   The implementation works by capturing failure in either the 'ExceptT' or in the underlying 'Parser'.
+--   The derived 'Alternative' instance will only recover from failures in the 'ExceptT'.
```
Comment (considerate) [inline src/Data/Aeson/Commit.hs:16]: I think the logic is reversed, we don't recover from the failures in the `ExceptT` right?

**Review by considerate — COMMENTED** [review]
I think this should be good to merge. See my comments below

File: aeson-commit.cabal (hunk 1 above)
Comment (jonascarpay) [reply aeson-commit.cabal]: Can you have multiple maintainers?

File: src/Data/Aeson/Commit.hs (hunk 2 above)
Comment (jonascarpay) [reply src/Data/Aeson/Commit.hs:16]: Yes we do? A `Right <parser failure>` counts as success to `Alternative`, so it won't recover from it.

File: aeson-commit.cabal (hunk 1 above)
Comment (jonascarpay) [reply aeson-commit.cabal]: I made myself maintainer and we share authorship

**Pull request merged**